types/react-big-calendar/index.d.ts

```diff
@@ -116,6 +117,12 @@ export interface HeaderProps {
     localizer: DateLocalizer;
 }
 
+export interface ResourceHeaderProps {
+    label: string;
```

Comment:
```suggestion
    label: React.ReactNode;
```

is equivalent to `PropTypes.node`. It does include `string`